### PR TITLE
JFG-1030: add JLoadProfileTps

### PR DESCRIPTION
**Pull request by anton-antonenko**

### CODE REVIEW

File: chassis/core/src/main/java/com/griddynamics/jagger/util/generators/WorkloadGenerator.java [hunk 1]
```diff
@@ -44,6 +48,15 @@ private static WorkloadClockConfiguration generateRps(JLoadProfileRps jLoadProfi
         return qpsClockConfiguration;
     }
 
+    private static WorkloadClockConfiguration generateTps(JLoadProfileTps jLoadProfile) {
+        TpsClockConfiguration tpsClockConfiguration = new TpsClockConfiguration();
+        tpsClockConfiguration.setValue(jLoadProfile.getTransactionsPerSecond());
+        tpsClockConfiguration.setWarmUpTime(jLoadProfile.getWarmUpTimeInSeconds());
```
Comment (oskliarov) [inline chassis/core/src/main/java/com/griddynamics/jagger/util/generators/WorkloadGenerator.java]: I think It's needed to convert to milliseconds.

Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/util/generators/WorkloadGenerator.java]: right, will change

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileTps.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+ * This type of load implements an exact number of transactions per second performed by Jagger.
+ * Available attributes:
+ *     - transactionsPerSecond - A goal number of transactions per second
+ *
+ * Optional attributes:
+ *     - maxLoadThreads - Maximum number of parallel threads allowed for load generation
+ *     - warmUpTimeInMilliseconds - Load will increase from 0 to @e transactionsPerSecond in this time
+ *
+ * @ingroup Main_Load_profiles_group
+ */
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileTps.java]: please add @@details section to documentation with the description of the difference between TPS and RPS (take from the old docu). add it to RPS docu as well

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/auxiliary/TransactionsPerSecond.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package com.griddynamics.jagger.user.test.configurations.load.auxiliary;
+
+/**
+ * The number of requests per second Jagger shall perform.
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/auxiliary/TransactionsPerSecond.java]: requests => transactions (request + response)

**Comment by dlatnikov** [comment]
please, add report with such load to the ticket and send me a link. thank you

**Pull request merged**